echo-model/src/main/java/com/netflix/spinnaker/echo/model/Trigger.java

```diff
@@ -160,6 +160,7 @@ public String toString() {
   List<String> expectedArtifactIds;
 
   // Configuration for plugin triggers
+  String pluginEventType;
```

Comment:
it's kind of awkward how `pluginEventType` is at the same level (the top) as all the plugin release fields in the Trigger model

Reply:
The trigger model is just awful. I think it makes sense to follow convention for now and we can tease `Trigger` into something else another time.